core/src/test/java/org/apache/iceberg/TestScanPlanningAndReporting.java

```diff
@@ -262,5 +263,12 @@ public ScanReport lastReport() {
       }
       return (ScanReport) reports.get(reports.size() - 1);
     }
+
+    public SnapshotReport lastSnapshotReport() {
+      if (reports.isEmpty()) {
+        return null;
+      }
+      return (SnapshotReport) reports.get(reports.size() - 1);
```

Comment:
Missing whitespace between the control flow statement and return.

Reply:
fixed